RoddyCore/src/de/dkfz/roddy/plugins/LibrariesFactory.groovy

```diff
@@ -45,6 +45,8 @@ public class LibrariesFactory extends Initializable {
     public static final String BUILDINFO_RUNTIME_JDKVERSION = "JDKVersion"
     public static final String BUILDINFO_RUNTIME_GROOVYVERSION = "GroovyVersion"
     public static final String BUILDINFO_RUNTIME_APIVERSION = "RoddyAPIVersion"
+    public static final String PRIMARY_ERRORS = "PRIMARY_ERRORS"  // Primary errors are "important"
+    public static final String SECONDARY_ERRORS = "SECONDARY_ERRORS" // Secondary errors could be "important" but are not checked in all cases.
```

Comment:
Intuitively (which might be wrong, of course) primary = initial, maybe even causal, and secondare is follow-up errors.